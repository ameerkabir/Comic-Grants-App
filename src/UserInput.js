import React from "react";
const UserInput = ({ postcode, search, setPostcode }) => {
  return (
    <div>
      <h1>Search Grants</h1>
      <form onSubmit={search}>
        <input
          type="text"
          value={postcode}
          onChange={setPostcode}
          placeholder={"postcode"}
          name="postcode"
          required
        />{" "}
        <button className="btn-primary">Search</button>
      </form>{" "}
    </div>
  );
};
export default UserInput;
