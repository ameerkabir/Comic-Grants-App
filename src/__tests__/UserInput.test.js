import React from 'react'
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import UserInput from '../UserInput'
Enzyme.configure({ adapter: new Adapter() });
describe('Component=>UserInput', ()=>{

    it('Should render correctly', ()=>{
        const wrapper = shallow(<UserInput/>)
        expect(wrapper).toMatchSnapshot()
    })
    
})