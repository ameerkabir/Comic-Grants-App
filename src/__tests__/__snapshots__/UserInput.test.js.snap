// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component=>UserInput Should render correctly 1`] = `
<div>
  <h1>
    Search Grants
  </h1>
  <form>
    <input
      name="postcode"
      placeholder="postcode"
      required={true}
      type="text"
    />
     
    <button
      className="btn-primary"
    >
      Search
    </button>
  </form>
   
</div>
`;
